script.sh

```diff
@@ -1,45 +0,0 @@
-echo "Installing shouston3's setup..."
```

Comment:
was this script difficult to maintain or just unused?

Reply:
I'm just not a fan of telling someone to run some script (which they might not check).
I thought forcing the user to see each command being run would be better - I also found myself opening the script to run each line instead.

Reply:
@samhstn makes sense. 👍